v5/securityandcompliancecenterapiv3/security_and_compliance_center_api_v3_integration_test.go

```diff
@@ -741,6 +741,7 @@ var _ = Describe(`SecurityAndComplianceCenterApiV3 Integration Tests`, func() {
 				ProfileName:        core.StringPtr("scc-go-sdk test_profile"),
 				ProfileDescription: core.StringPtr("test_description1"),
 				ProfileType:        core.StringPtr("custom"),
+				ProfileVersion:     core.StringPtr("0.0.1"),
```

Comment:
Pretty sure by default Versions are 1.0.0

Reply:
the default version is actually 0.0.1, but adding ProfileVersion allows you to change it to a specific version in ReplaceProfileOptions